### DISCUSSION TITLE
sink.properties.columns config issue

### DISCUSSION THREAD

**Discussion by ChenShuai1981** [apache/doris#45615] [A - General / Q&A]
flink doris connector 什么情况需要用到 'sink.properties.columns' 配置参数？在flink sql中不是有定义sink doris的table列么，为什么还需要再在这个参数中申明需要插入哪些字段呢？即使是在部分列更新场景也可以从flink sql table中获取字段名呀，例如：
CREATE TABLE flink_doris_sink (
    name STRING,
    age INT,
    price DECIMAL(5,2),
    sale DOUBLE
    ) 
    WITH (
      'connector' = 'doris',
      'fenodes' = 'FE_IP:HTTP_PORT',
      'table.identifier' = 'db.table',
      'username' = 'root',
      'password' = 'password',
      'sink.label-prefix' = 'doris_label'，
      'sink.properties.partial_columns' = 'true',
      ‘sink.properties.columns’=‘name,age,price,sale’
);

**Comment by ixzc** [comment]
中文问题推荐在论坛提问
https://ask.selectdb.com/
